### PR TITLE
build: create script for setting up RBE in local dev environment

### PR DESCRIPTION
**Pull request by josephperrott**

### CODE REVIEW

File: scripts/local-dev/setup-rbe-for-local-dev.sh [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# A script for automatically configuring a user's local dev
+# environment to use Remote Build Execution.
+
+# Determine the root directory of the Angular github repo.
+GITHUB_DIRECTORY_ROOT=$(git rev-parse --show-toplevel 2> /dev/null);
+if [[ $? -ne 0 ]]; then
+  echo "This command must be run from within the angular repository."
```
Comment (devversion) [inline scripts/local-dev/setup-rbe-for-local-dev.sh]:
```suggestion
  echo "This command must be run from within the cloned \"angular/components\" repository."
```

File: scripts/local-dev/setup-rbe-for-local-dev.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+# Confirm gcloud installed and available as a command.
+if [ ! -x "$(command -v gcloud)" ]; then
+  echo "gcloud command is not available.  Please install gcloud before continuing."
+  exit 1;
+fi
+
+# Confirm the parameter provided to the script is a directory
+if [[ ! -d $1 ]]; then
+  echo "The paramter provided is not a directory";
```
Comment (devversion) [inline scripts/local-dev/setup-rbe-for-local-dev.sh]: There is a typo for `parameter` here. How about showing the usage instead? Maybe also expand on why we need to even specify a directory.

```suggestion
  echo "Invalid command syntax. Usage: ./script.sh <ServiceAccountKeyLocation>";
```

File: scripts/local-dev/setup-rbe-for-local-dev.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  exit 1;
+fi
+ABSOLUTE_DIRECTORY_PATH=$(readlink -f $1)
```
Comment (devversion) [inline scripts/local-dev/setup-rbe-for-local-dev.sh]: Nit: For clarity I think the variable name should be named differently. Absolute directory path only describes the type but not what the variable contains.

Comment (josephperrott) [reply scripts/local-dev/setup-rbe-for-local-dev.sh]: Yeah, I can see that, I tried something else.  I can't come up with a good name so if you have any ideas let me know.

File: scripts/local-dev/setup-rbe.sh [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+# Determine the root directory of the Angular github repo.
+GITHUB_DIRECTORY_ROOT=$(git rev-parse --show-toplevel 2> /dev/null);
+if [[ $? -ne 0 ]]; then
+  echo "This command must be run from within the cloned \"angular/components\" repository."
+  exit 1;
+fi
+
+# Confirm gcloud installed and available as a command.
+if [ ! -x "$(command -v gcloud)" ]; then
+  echo "gcloud command is not available.  Please install gcloud before continuing."
```
Comment (jelbourn) [inline scripts/local-dev/setup-rbe.sh]: Could this script install `gcloud`?

File: scripts/local-dev/setup-rbe-for-local-dev.sh (hunk 3 above)
Comment (devversion) [reply scripts/local-dev/setup-rbe-for-local-dev.sh]: `credentialsDirectory`?

File: scripts/local-dev/setup-rbe.sh (hunk 4 above)
Comment (josephperrott) [reply scripts/local-dev/setup-rbe.sh]: We can't install `gcloud` because dependent on your platform we have to install it differently.

**Pull request merged**